### DISCUSSION TITLE
[iOS] Weird UICollectionView behavior in pinned repositories

### DISCUSSION THREAD

**Discussion by onl1ner** [community/community#3801] [Mobile] [answered]
Not sure if this bug is only reproducible in my environment but I am having this weird scrolling behavior of UICollectionView in pinned repositories in profile page.

**Steps to reproduce:** 
1. Open GitHub application
2. Go to Profile tab
3. Scroll collection in pinned section

**Specs**:

- **Device**: iPhone 11 Pro Max
- **iOS**: 14.2
- **GitHub Mobile Version**:  1.10.0

https://user-images.githubusercontent.com/45308586/115104267-2bd26500-9f79-11eb-84b1-e33b638f1bb4.mov

**Comment by eliperkins** [comment, marked as answer]
Thanks for reporting this @onl1ner!

We're tracking this bug in our internal issues, and we're hoping to have a fix for this soon. Stay tuned!
